packages/connectors/src/Connector.Shared.ts

```diff
@@ -69,11 +88,13 @@ export interface ChiliResponse extends ChiliBody {
 }
 
 export type ConnectorConfigValueType = 'text' | 'boolean';
+export type ConnectorConfigContextType = 'query' | 'upload';
 
 export interface ConnectorConfigValue {
     readonly name: string;
     readonly displayName: string;
     readonly type: ConnectorConfigValueType;
+    readonly context: ConnectorConfigContextType[];
```

Comment:
Should be optional to avoid breaking changes of existing connectors

Reply:
flutter will handle this, if it's undefined they will treat it as browse. I'm wondering if this can be a breaking change since it's inside the connector package?

Reply:
It's a breaking change from API definition - it enforce existing connectors to define context as empty array in their ConnectorConfigValues

Reply:
I mean it's a breaking change from this interface definition, should be define following in such case

```suggestion
readonly context?: ConnectorConfigContextType[];
```